modules/ROOT/pages/type-definitions/directives/index.adoc

```diff
@@ -1,11 +1,182 @@
 [[directives]]
 :description: This page lists all directives available in the Neo4j GraphQL Library.
+
 = Directives
 :page-aliases: directives.adoc
 
 The Neo4j GraphQL Library provides the following directives to be used whilst defining types:
 
-[cols="1,3"]
+== Database mapping
+
+.Database mapping directives
```

Comment:
I think you are not required to name the tables, so you can avoid the repetition with the name of the section